exercises/practice/acronym/test-words.8th

```diff
@@ -52,3 +66,35 @@ needs console/loaded
    then 
 ;
 
+\ compare arrays by testing elements with string equality
+: test_eqa \ s x w -- | s w x -- 
+   -test-count
+   isword? !if swap then
+   w:exec
+   ' s:= a:= nip nip if
+    con:green con:onBlack . space " ... OK" . con:white con:onBlack cr
+   else
+    con:red con:onBlack . space " ... FAIL" . con:white con:onBlack cr
+   then 
+;
+
+: test_map_neq \ m m -- 
+   -test-count
+   isword? !if swap then
+   a:new swap 0 swap a:! 0 a:@ nip w:exec
+   ' n:= m:= !if
+    2drop
+    con:green con:onBlack . space " ... OK" . con:white con:onBlack cr
+   else
+    2drop
+    con:red con:onBlack . space " ... FAIL" . con:white con:onBlack cr
+   then 
+;
+
+( test-count @ 0 n:= 
+  !if 
+  con:red con:onBlack
+  "FAIL - not all tests completed" . 
```

Comment:
To trigger the test runner, we need dots here.
```suggestion
  "... FAIL - not all tests completed" . 
```

Reply:
Yes, I was wondering about that